src/components/Recipes/recipeCard/index.jsx

```diff
@@ -18,22 +24,33 @@ export const RecipeCard = ({ recipe, mode = "all-recipes", page, category }) =>
             alt={recipe.name}
             className="w-full h-48 object-cover"
           />
-          <div className="p-4 flex flex-col justify-between h-full">
+          <div className="p-4 flex flex-col justify-between h-full ">
             <h2 className="text-2xl font-semibold mb-2">{recipe.name}</h2>
             <div>
               <div>
-              <h2 className="font-bold inline">type </h2>
-              <p className="inline">
-                {recipe.kosherType}
-              </p>
+                <h2 className="font-bold inline">type </h2>
+                <p className="inline">{recipe.kosherType}</p>
               </div>
               <div>
-                <div onClick={()=>navigate(`/user/${recipe?.userOwner?.id}`,{state:{username:recipe?.userOwner?.username}})}>
-                <h2 className="font-bold text-lg inline">By </h2>
-                <p className="mb-2 text-gray-800 hover:text-blue-500 inline">
-                  {recipe?.userOwner?.username}
-                </p>
-              </div>
+                <div
+                  onClick={() =>
+                    navigate(`/user/${recipe?.userOwner?.id}`, {
+                      state: { username: recipe?.userOwner?.username },
+                    })
+                  }
+                >
+                  <h2 className="font-bold text-lg inline">By </h2>
+                  <p className="mb-2 text-gray-800 hover:text-blue-500 inline">
+                    {recipe?.userOwner?.username}
+                  </p>
```

Comment:
If you use destructuring as explained in the previous comment you will avoid this `x?y?z` logic